sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -162,7 +162,8 @@ statement
         tableIdentifier partitionSpec? describeColName?                #describeTable
     | REFRESH TABLE tableIdentifier                                    #refreshTable
     | REFRESH (STRING | .*?)                                           #refreshResource
-    | CACHE LAZY? TABLE tableIdentifier (AS? query)?                   #cacheTable
+    | CACHE LAZY? storageLevel=identifier? TABLE
+        tableIdentifier (AS? query)?                                   #cacheTable
```

Comment:
As another syntax option, how about this?
```
CACHE TABLE testTable OPTIONS ('storageLevel'='disk_only')
UNCACHE TABLE testTable OPTION ('cascade'='true', 'blocking'='true')
```

Reply:
Thanks @maropu, I changed to option.